packages/rspack/src/Compilation.ts

```diff
@@ -103,11 +103,22 @@ export class Compilation {
 	logging: Map<string, LogEntry[]>;
 	name?: string;
 	childrenCounters: Record<string, number> = {};
+	/**
+	 * Note: This is not a webpack public API, maybe removed in future.
```

Comment:
nitpick: comment on why is this needed, and the requirement for removal. Having "maybe" does not communicate well.

Reply:
Leaving a phrase like "do not use" maybe too aggressive, so I used "maybe" to make the tone softer. What else would you recommend?